src/rpc/mempool.cpp

```diff
@@ -98,7 +98,15 @@ static RPCHelpMan testmempoolaccept()
                 },
             },
             {"maxfeerate", RPCArg::Type::AMOUNT, RPCArg::Default{FormatMoney(DEFAULT_MAX_RAW_TX_FEE_RATE.GetFeePerK())},
-             "Reject transactions whose fee rate is higher than the specified value, expressed in " + CURRENCY_UNIT + "/kvB\n"},
+             "(deprecated. Use \"options\" instead) Reject transactions whose fee rate is higher than the specified value, expressed in " + CURRENCY_UNIT + "/kvB\n"},
```

Comment:
Replacing the option is fine. Only need to tolerate the old param type.

Might need to pull in 09964963a1f7748fc1abc9471cbae8dab1061eb1

Reply:
Are you suggesting something like the `dummy` argument in `getbalance()` ?

```c++
RPCHelpMan getbalance()
{
  ....
  {"dummy", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "Remains for backward compatibility. Must be excluded or set to \"*\"."},
```

Reply:
No, more like this:

```c++
{"options|maxfeerate", {RPCArg::Type::OBJ, RPCArg::Type::AMOUNT}, RPCArg::Optional::OMITTED_NAMED_ARG, "",
```

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25577/commits/841c77b1831f81c3ebd26d110adba7815b10a8d4.